packages/sigthief/PKGBUILD

```diff
@@ -0,0 +1,29 @@
+# This file is part of BlackArch Linux ( http://blackarch.org ).
+# See COPYING for license details.
+
+pkgname='sigthief'
```

Comment:
No need for quotes. This might actually be a bad thing.

Reply:
i will change it